src/OSPSuite.Assets/UIConstants.cs

```diff
@@ -236,17 +241,19 @@ public static string DoYouWantToDeleteDirectory(string newDirectoryName)
          return $"Do you want to delete the directory '{newDirectoryName}' and continue?";
       }
 
-      public static string ExceptionViewDescription(string supportEmail)
+      public static string ExceptionViewDescription(string issueTrackerUrl)
       {
          var sb = new StringBuilder();
          sb.AppendLine("oops...something went terribly wrong.");
          sb.AppendLine();
-         sb.AppendLine($"To best address the error, please send us an email at <B>{supportEmail}</B> with:");
-         sb.AppendLine("    1 - the information below");
-         sb.AppendLine("    2 - the steps you took prior to the problem emerging");
-         sb.AppendLine("    3 - if possible, attach your project file to the email (do not send confidential information by email).");
+         sb.AppendLine("To best address the error, please enter an issue in our issue tracker:");
+         sb.AppendLine($"    1 - Visit <b>{issueTrackerUrl}</b> or click on the link below");
+         sb.AppendLine("    2 - Click on the <b>New Issue</b> button");
+         sb.AppendLine($"    3 - Copy the information below by using the <b>{CopyToClipboard}</b> button and paste it in the issue description");
+         sb.AppendLine("    2 - Describe the steps you took prior to the problem emerging");
+         sb.AppendLine("    3 - if possible, attach your project file to the issue (do not attach confidential information).");
```

Comment:
1 2 3 2 3 or 1 2 3 4 5? ;-)
Also I would put "_Describe the steps..._" before "_Copy the information..._"